src/views/DefineCompany.vue

```diff
@@ -28,27 +28,56 @@
         </div>
       </div>
     </v-card>
+    </section>
+    <p></p>
```

Comment:
Empty <p> tag here intentional?

Reply:
Agreed. Please use margin or padding instead. (Could even use class="mt-4" in next section tag.)

Reply:
No it was not intentional. This was supposed to be cleaned up. Will add the style

Reply:
Sounds good!